src/o365/graph/commands/schemaextension/schemaextension-set.ts

```diff
@@ -0,0 +1,246 @@
+import commands from '../../commands';
+import request from '../../../../request';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import GraphCommand from '../../../base/GraphCommand';
+import Utils from '../../../../Utils';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  description: string;
```

Comment:
Properties that are optional, such as `description` should be designated as `description?: string` to make it clear that they are optional and could be missing on the passed properties.